docs/user/concepts.md

```diff
@@ -159,6 +160,22 @@ To further simplify device registration, Provisioning Tokens can be created to a
 devices to automatically connect to a team without having to manually register them first.
 The token can also be configured to assign a device directly to a Node-RED instance within the team.
 
+### Device Actions
```

Comment:
Not 100% sure if this warrants being a top-level concept. It feels more like a feature of devices that should be in the device specific docs.

Reply:
Unfortunately, there are no /user/device specific docs and /device-agent docs are about the agent and its installation/registration. 

possible options:
1. remove this all together
   1. viable - there is no counterpart "instance actions" in the docs (only a smattering of "... requires a restart of the Instance")
2. keep it and add a follow up task to generate full user/device docs and move it
3. generate full user/device docs as part of this PR
4. other?

^ @knolleary thoughts?

Reply:
I would pull it out of this PR. The docs need more work than we should try to contain in this one PR. In the short term, this feature is reasonably self-explanatory in the UI so doesn't need lots of accompany documentation.

Reply:
resolved in [8e83791](https://github.com/FlowFuse/flowfuse/pull/3548/commits/8e837918fab03c127f20f5b7d771edcacdcd08b6), task here: https://github.com/FlowFuse/flowfuse/issues/3566